client/src/Tools/FetchData/additionalToolsForResults.js

```diff
@@ -0,0 +1,10 @@
+export function removeNullsFromArray(array = []) {
+  let newArray = [];
+  array.forEach((element) => {
+    if (element !== null) {
```

Comment:
if(element) shuold do the trick, or you could just return a filtered array instead of forEach (return array.filter(element etc)

Reply:
I had a feeling there must be built-in JS method for such behaviour :D